src/Constant.js

```diff
@@ -56,7 +56,7 @@ export const drawTools = {
     vertical:   { id: 'vertical',    text: t.translate('Vertical [num]'), icon: DrawToolsVerticalIcon },
 };
 
-export const IndicatorsTree = [
+export const IndicatorsTree = () => [
```

Comment:
can we change this to camel case? `getIndicatorsTree`

Reply:
@balakrishna-binary thanks, just rename it